cmd/okctl/create.go

```diff
@@ -5,6 +5,8 @@ import (
 	"path"
 	"regexp"
 
+	"github.com/oslokommune/okctl/pkg/config/constant"
```

Comment:
When I see it like this, I get it! It flows nicely ❤️

Reply:
woooh im glad!